### ISSUE TITLE
No warning should be issued when a package is required by two packages, one requiring a specific version and the other not

### ISSUE THREAD

**Issue by eli-b** [naiquevin/pipdeptree#48]
I'm getting the following warning:

> Warning!!! Possibly conflicting dependencies found:
* cryptography==1.4
 - setuptools [required: >=11.3, installed: \<unknown>]
* zope.interface==4.2.0
 - setuptools [required: None, installed: \<unknown>]
* repoze.who==2.3
 - setuptools [required: None, installed: \<unknown>]
* protobuf==3.0.0b2.post2
 - setuptools [required: None, installed: \<unknown>]

But I think no warning should be issued here as all packages but the first require just any version of setuptools.

**Comment by jimbocoder** [comment]
Also ran into this issue on my first day with the tool. Easy fix, no controversy, right? I could do a PR if there's still interest.

**Comment by szepeviktor** [comment]
Do I have the same issue?

```
Warning!!! Possibly conflicting dependencies found:
* cryptography==1.5
 - setuptools [required: >=11.3, installed: 27.1.2]
```

**Comment by eli-b** [comment]
This looks like a different issue. Your installed version is indeed >= 11.3.
Do you use the latest version of pipdeptree? Many issues were fixed in it.

בתאריך יום א׳, 16 באוק׳ 2016, 17:29, מאת Viktor Szépe ‏<
notifications@github.com>:

**Comment by szepeviktor** [comment]
I have 0.7.0 and of course setuptools 27.1.2 is installed.

**Comment by naiquevin** [comment]
The original bug has been fixed in 0.7.0. I've pushed a commit for #55 and planning to release 0.8.0 in a short while. Closing this ticket.
